doc/ale-erlang.txt

```diff
@@ -13,6 +13,14 @@ g:ale_erlang_dialyzer_executable             *g:ale_erlang_dialyzer_executable*
   This variable can be changed to specify the dialyzer executable.
 
 
+g:ale_erlang_dialyzer_options                   *g:ale_erlang_dialyzer_options*
+                                                *b:ale_erlang_dialyzer_options*
+  Type: |String|
+  Default: `'-Wunmatched_returns -Werror_handling -Wrace_conditions -Wunderspec %s'`
```

Comment:
Any particular reason to have %s as part of the options instead of harcoded in the command function like all other linters?

Reply:
It doesn't have to be, indeed.

I removed it